frontend/src/pages/recruitment/Final.js

```diff
@@ -0,0 +1,126 @@
+    end_date: new Date(),
+  });
+  const dispatch = useDispatch();
+
+  React.useEffect(() => {
+    const loadData = async () => {
+      const { ok, data } = await getApplications(state.start_date, state.end_date, false, "Final");
+      if (ok) {
+        const applications = data.applications.map((app) => ({
+          ...app,
+
+          role: app.role.name,
+        }));
+        setState((prev_state) => ({
+          ...prev_state,
+          loading: false,
+          applications,
+        }));
+      } else {
+        setState((prev_state) => ({
+          ...prev_state,
+          loading: false,
+        }));
+      }
+    };
+    if (state.loading) {
+      loadData();
+    }
+    const loadReviews = async () => {
+      const reviews = state.applications.map((x) => x);
+      console.log(reviews.length);
+      let i;
+      for (i = 0; i < reviews.length; i++) {
+        const curr = i;
+        const review_list = await getApplicationReviews(reviews[curr]._id);
+        reviews[curr][
+          "resume_rating"
+        ] = `${review_list.data.reviews[0].reviewer.name}: ${review_list.data.reviews[0].rating}/5`;
+        reviews[curr][
+          "phone_rating"
+        ] = `${review_list.data.reviews[1].reviewer.name}: ${review_list.data.reviews[0].rating}/5`;
+        reviews[curr][
+          "interview_rating"
+        ] = `${review_list.data.reviews[2].reviewer.name}: ${review_list.data.reviews[0].rating}/5`;
+      }
+      setState((prev_state) => ({
+        ...prev_state,
+        loading: false,
+        reviews,
+      }));
+    };
+    loadReviews();
+  }, [state.applications, state.loading, state.start_date, state.end_date, dispatch]);
+  return (
+    <>
+      <Helmet>
+        <title>Final Applications — TSE Oktavian</title>
+      </Helmet>
+      <PageContainer>
+        <LoadingContainer loading={state.loading}>
+          <MuiPickersUtilsProvider utils={DateFnsUtils} />
+          <MaterialTable
+            icons={TableIcons}
+            actions={[
+              {
+                icon: function visibility() {
+                  return <Visibility />;
+                },
+                tooltip: "View Application",
+                onClick: (event, row) => {
+                  const origin = `${window.location.protocol}//${window.location.hostname}${
+                    window.location.port ? `:${window.location.port}` : ""
+                  }`;
+                  window.open(`${origin}/recruitment/application/${row._id}`);
+                },
+              },
+            ]}
+            options={{
+              paging: true,
+              pageSize: 10,
+              emptyRowsWhenPaging: true,
+              pageSizeOptions: [10, 20, 50, 100],
+            }}
+            columns={[
+              { title: "Name", field: "name" },
+              { title: "Email", field: "email" },
+              { title: "Position", field: "resume" },
```

Comment:
I think you might've forgotten to change the title of this column to resume or something similar. Since there is already a column called resume (the resume stage), maybe this can be named something like "resume link"

Reply:
I also think this still needs to be updated since the column seems to hold the link to a resume